tests_tf/test_attacks_tf.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np
+import unittest
+import tensorflow as tf
+import tensorflow.contrib.slim as slim
+
+from cleverhans import attacks_tf
+
+class TestAttacksTF(unittest.TestCase):
+    def test_jsma_batch_with_feed(self):
+        with tf.Session() as sess:
+            X = np.random.rand(1, 13)
+
+            # construct a simple graph that will require extra placeholders
+            x = tf.placeholder('float', shape=(None, 13))
+            b = tf.placeholder('bool')
+            logits = slim.dropout(slim.fully_connected(x, 10),
```

Comment:
Could you do this without adding a dependency on slim? Since it's in the `tensorflow.contrib`, its API is more likely to change quickly and it's not necessary to test the logic here.